core/primitives-core/src/version.rs

```diff
@@ -243,7 +243,7 @@ const STABLE_PROTOCOL_VERSION: ProtocolVersion = 66;
 /// Largest protocol version supported by the current binary.
 pub const PROTOCOL_VERSION: ProtocolVersion = if cfg!(feature = "statelessnet_protocol") {
     // Current StatelessNet protocol version.
-    84
+    85
```

Comment:
Could you add a quick comment below `ProtocolFeature::StatelessnetShuffleShardAssignmentsForChunkProducers => 84,` saying the next protocol version just implements the hard limit and we should use 86 for the next feature?

Reply:
Just to make sure I understand, why is there no need to check for it? Is it because it's checked in the runtime params? 

I have to say it is rather confusing to see bumped protocol version without corresponding protocol feature. I would recommend adding it just for clarity or adding a comment somewhere.

Reply:
> Just to make sure I understand, why is there no need to check for it? Is it because it's checked in the runtime params?

Yes, for previous protocol versions the limit is 9999999999 (+inf), so it will never be triggered. The limit becomes 4MB at protocol 85 and it's defined in the runtime params, so there's no need for protocol feature checks.

> I have to say it is rather confusing to see bumped protocol version without corresponding protocol feature. I would recommend adding it just for clarity or adding a comment somewhere.

I will add one :+1: